README.md

```diff
@@ -10,93 +10,168 @@ providing QUnit-specific wrappers around the helpers contained in
 
 ## Usage
 
-### Setting the resolver
+- [Acceptance Tests](#acceptance-tests)
+- [Component Unit Tests](#component-unit-tests)
+- [Component Integration Tests](#component-integration-tests)
+- [Other Tests](#other-tests)
+- [Ember Data Tests](#ember-data-tests)
+
+### Acceptance Tests
```

Comment:
This (acceptance tests) doesn't really have to do with ember-qunit.  It should absolutely be documented, but not in this repo.